.git-blame-ignore-revs

```diff
@@ -0,0 +1,17 @@
+##
+# All "yarn fix" commit revs should be included in this file.
+#
+# Add to your local configuration:
+#   git config --global blame.ignoreRevsFile .git-blame-ignore-revs
```

Comment:
Doing it globally messes up other repos, since Git expects this file to exist everywhere. Making the config local to the repo is the only safe option.

Reply:
It was defined this way in the task requirements, but you are very correct about the breakage here. I'll change the comment and also change the requirement in the task